src/Pages/CloudAccountsPage/CloudAccountsPage.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { useEffect } from 'react';
+import { CloudAccountsTable } from '../../Components/CloudAccounts/CloudAccountsTable';
+import { Content, PageSection } from '@patternfly/react-core';
+import { Section } from '@redhat-cloud-services/frontend-components/Section';
+import { PageHeader } from '@redhat-cloud-services/frontend-components/PageHeader';
+import { CloudAccountsPagination } from '../../Components/CloudAccounts/CloudAccountsPagination';
+import { CloudAccountsToolbar } from '../../Components/CloudAccounts/CloudAccountsToolbar';
+import { useCloudAccounts } from '../../hooks/api/useCloudAccounts';
+import { Unavailable } from '@redhat-cloud-services/frontend-components/Unavailable';
+import { Navigate } from 'react-router-dom';
+import { Loading } from '../../Components/util/Loading';
+import { useRbacPermission } from '../../hooks/util/useRbacPermissions';
+import { Paths } from '../../utils/routing';
+import { NoCloudAccounts } from '../../Components/CloudAccounts/NoCloudAccounts';
+import { useQueryParamInformedAtom } from '../../hooks/util/useQueryParam';
+import { CloudAccountsPaginationData } from '../../state/cloudAccounts';
+
+export const CloudAccountsPage = () => {
+  const [pagination, setPagination] = useQueryParamInformedAtom(
+    CloudAccountsPaginationData,
+    'pagination'
+  );
+
+  const { page, perPage } = pagination;
+
+  const {
+    data: cloudAccountsResponse,
+    isError: isCloudAccountsError,
+    isLoading: areCloudAccountsLoading,
+  } = useCloudAccounts({ limit: perPage, offset: (page - 1) * perPage });
+
+  const accounts = cloudAccountsResponse?.body ?? [];
+  const hasAccounts = accounts.length > 0;
+  useEffect(() => {
+    if (cloudAccountsResponse?.pagination) {
+      setPagination({
+        ...pagination,
+        itemCount: cloudAccountsResponse.pagination.total,
+      });
+    }
+  }, [cloudAccountsResponse?.pagination?.total]);
```

Comment:
Cleanup: The pagination `itemCount` is also being set in the cloud accounts table, it probably only needs to be set in one place